esp-hal-common/src/uart.rs

```diff
@@ -483,6 +489,12 @@ where
         serial.change_parity(config.parity);
         serial.change_stop_bits(config.stop_bits);
         serial.change_baud(config.baudrate, clocks);
+        // Should this be done ? Or configuration option? Setting it stops uart from
```

Comment:
it's probably fine to set it

Reply:
okay then I will keep it and update description of pull request. Also will remove comment questions

Reply:
Done